README.md

```diff
@@ -313,6 +314,30 @@ The default `interval` is `50ms`. However it will run your callback immediately
 on the next tick of the event loop (in a `setTimeout`) before starting the
 intervals.
 
```

Comment:
Let's update the docs here to document `renderIntoDocument` and `clearDocument` (in that order). Then update the `fireEvent` docs to explain why it's important to use those helpers.

Reply:
oh i just added this, but added them below `render` and before `Simulate`